LAMPAPI/delete.php

```diff
@@ -10,16 +10,17 @@
 	else
 	{
         //Look for user in the database
-        $stmt = $conn->prepare("SELECT * from Contacts where Email like ? and ID=?");
+        $stmt = $conn->prepare("SELECT * from Contacts where Email like ? and ID=? and UserID=?");//added UserID when searching for the contact
 		$email = $inData["Email"];
-        $stmt->bind_param("si", $email, $inData["ID"]);
+        $stmt->bind_param("sii", $email, $inData["ID"], $inData["UserID"]);
```

Comment:
Since email is unique, we dont really need the ID number of the contact being deleted after u delete ID we should be fine

Reply:
I removed the ID number from the delete.php